app/views/unattended/provisioning_templates/snippet/puppet.conf.erb

```diff
@@ -12,7 +12,7 @@ description: |
   os_family = @host.operatingsystem.family
   os_name   = @host.operatingsystem.name
 
-  aio_enabled = host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
+  aio_enabled = host_param_true?('enable-puppetlabs-repo') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
```

Comment:
If the parameter should be _version non-specific flag_, we could name it just simply `enable-puppet`, right?

Reply:
I certainly wouldn't mind that, but I was trying to keep the patch to the minimum size without too much deeper impact.

With puppet also packaged in fedora, I'm not sure that `enable-puppet` would clearly indicate that this also creates external repos.

Reply:
OK, make sense